StortSpelProjekt/Game/src/main.cpp

```diff
@@ -414,7 +415,7 @@ Scene* GameScene(SceneManager* sm)
 
     EnemyFactory::GetInstance().DefineEnemy("enemyZombie", &zombie);
     EnemyFactory::GetInstance().DefineEnemy("enemySpider", &spider);
-    EnemyFactory::GetInstance().DefineEnemy("enemyDemon", &rangedDemon);
```

Comment:
Changed define enemy causing no demons to be spawned

Reply:
That's a biiig oopsie 😓